ui/src/components/pages/TaskListScreen.js

```diff
@@ -38,9 +42,11 @@ const TasksScreen = (props) => {
             <Button
               className="task-Button"
               key={task.taskId}
+              onDelete={this.handleDelete} 
```

Comment:
using modern syntax we can avoid using the _this_ keyword entirely here. however, we would need to send the taskId prop down so the delete button knows what the current task's ID is
nitpicking: the common format is on-Action-Handler
```suggestion
              onDelete={onDeleteHandler} 
              taskId={task.taskId}
```